projects/plugins/crm/modules/portal/sass/jpcrm-public-portal.scss

```
@@ -719,3 +719,11 @@ i.icon.child:before {
   display: inline-block;
   text-align: center;
 }
+
+.jpcrm-client-portal-admin-banner {
+	padding: 0 20px 20px 20px;
```

I believe technically you could do the following, by grouping left and right padding together as they are the same:

```suggestion
	padding: 0 20px 20px;
```
Not a blocker at all though :D

---

This is true; I personally prefer being explicit in using the full expansion for readability/easy adjustment. I assumed the SASS compiler would auto-adjust this but I just checked and it doesn't. 🤔

Either way, I think I'll leave it.